ruby/ql/test/library-tests/dataflow/summaries/Summaries.ql

```diff
@@ -101,23 +101,21 @@ private class TypeFromModel extends ModelInput::TypeModelCsv {
   override predicate row(string row) {
     row =
       [
-        "test;FooOrBar;;;Member[Foo].Instance", //
-        "test;FooOrBar;;;Member[Bar].Instance", //
-        "test;FooOrBar;test;FooOrBar;Method[next].ReturnValue",
+        "~FooOrBar;Foo;", //
```

Comment:
Could you remind me what the `~` does?

Reply:
It's the convention for ensuring that we don't clash with a built-in type. The `~` character itself doesn't do anything special. If we simply called the type `ForOrBar` it would be also interpreted to mean instances of `FooOrBar`.